src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.util.Colors;
+
+public class Circle extends Figure {
+
+    private int rad;
+    private Colors color;
+
+    public Circle() {
+        this.rad = getRandomNum();
+    }
```

Comment:
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#you-can-pass-random-values-to-the-constructor-of-a-figure-instead-of-generating-them-inside-figure-classes

Reply:
Apply this to rest of the figures as well - pass color value to `super()`